src/graphics/components/movie/storyboard.cpp

```diff
@@ -6,12 +6,122 @@
 
 #include "graphics/mesh.hpp"
 
+#include <eigen3/Eigen/Dense>
 #include <imgui.h>
 
+#include "path.hpp"
+
 namespace tomovis {
 
 Animation::Animation(float at, float duration) : at_(at), duration_(duration) {}
 
+MoveAlongPath::MoveAlongPath(std::vector<float> time_points, Path3 const& path,
+                             glm::vec3& target)
+    : Animation(time_points[0],
+                time_points[time_points.size() - 1] - time_points[0]),
```

Comment:
This can segfault when `time_points` has no entries. Any good way to avoid that?